src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const getSalaryNumber = (str) =>
+  Number(str.trim().split(',').join('').slice(1));
+
+document.querySelector('thead').addEventListener('click', function(e) {
+  const clicked = e.target;
+  const type = clicked.textContent;
+  const index = [...clicked.closest('tr').children].indexOf(clicked);
+
+  const tableBody = document.querySelector('tbody');
+  const tableData = [...tableBody.children];
+
+  tableData.sort((a, b) => {
+    const firstText = a.children[index].textContent;
+    const secondText = b.children[index].textContent;
+
+    switch (type) {
+      case 'Position':
+      case 'Name':
+        return (firstText.localeCompare(secondText));
+      case 'Age':
+        return firstText - secondText;
+      case 'Salary':
+        return getSalaryNumber(firstText) - getSalaryNumber(secondText);
+    };
+  });
```

Comment:
please move this logic to separate function, or even create function that will return you result of sorting